modules/shared/config/config.go

```diff
@@ -56,6 +56,15 @@ type Config struct {
 	AllowedNamespaces []string `mapstructure:"allowed-namespaces"`
 	KubeconfigPath    string   `mapstructure:"kubeconfig"`
 
+	// OTel options
+	OTel struct {
```

Comment:
I think each component (Dashboard, Cluster API, Cluster Agent) should have its own instance of the OTel options so we can keep them independent. Soon we will probably split up the shared config into separate configs so having separate otel configs will help that.